src/Resources/config/core.php

```diff
@@ -46,7 +46,6 @@
         ->args([
             new ReferenceConfigurator('sonata.block.loader.chain'),
             new ReferenceConfigurator('sonata.block.manager'),
-            '%sonata_block.cache_blocks%',
             (new ReferenceConfigurator('logger'))->nullOnInvalid(),
```

Comment:
should we keep the logger as argument ? You'll end with a non solvable deprecation isnt it ?

Reply:
having logger as 3rd argument is the new signature and that's fine